packages/snippet-bot/test/snippet-bot.ts

```diff
@@ -542,6 +542,64 @@ describe('snippet-bot', () => {
       requests.done();
       diffRequests.done();
     });
+
+    it('gives fyi message for removing frozen region tag', async () => {
+      sandbox.restore();
```

Comment:
Given that most of the logic is added in `violations.ts` which has already been broken out into a helper, I wonder if it would be better to add some tests for `violations.ts`, vs., end to end tests using `nock`?

Reply:
I added unit test for violations.ts.